### PR TITLE
docs: Add Eduardo Rodrigues to dev team

### PR DESCRIPTION
**Pull request by matthewfeickert**
To give credit where credit is due for ensuring development continues, add Eduardo Rodrigues to the credits of developers.

**Squash and merge commit message**
```
* Add Eduardo Rodrigues to development team listing.
   - Update Zenodo citation metadata, recommended citation, and setup.cfg.
```

### CODE REVIEW

File: .zenodo.json [hunk 1]
```diff
@@ -14,6 +14,11 @@
             "affiliation": "University of Wisconsin-Madison",
             "name": "Matthew Feickert",
             "orcid": "0000-0003-4124-7862"
+        },
+        {
+            "affiliation": "University of Liverpool",
+            "name": "Eduardo Rodrigues",
+            "orcid": "0000-XXX-YYY-ZZZ"
```
Comment (matthewfeickert) [inline .zenodo.json:21]: @eduardo-rodrigues do you have an [ORCID](https://orcid.org/)? I didn't see if on your profile after a very quick check.

Comment (eduardo-rodrigues) [inline .zenodo.json:21]:
```suggestion
            "orcid": "0000-0003-2846-7625"
```

Comment (eduardo-rodrigues) [reply .zenodo.json:21]: Hi. Yep, see for example https://github.com/scikit-hep/particle/blob/master/.zenodo.json or similarly in the scikit-hep metapackage.

**Comment by eduardo-rodrigues** [comment]
Thank you very much for this!

**Pull request merged**